### ISSUE TITLE
The autocomplete not working after Enter.

### ISSUE THREAD

**Issue by Andronachi-Marian** [atom/autocomplete-css#108]
*Edited by @rsese to flesh out the issue template*

### Prerequisites

* [X] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in Safe Mode: http://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode
    * Followed all applicable steps in the debugging guide: http://flight-manual.atom.io/hacking-atom/sections/debugging/
    * Checked the FAQs on the message board for common solutions: https://discuss.atom.io/c/faq
    * Checked that your issue isn't already filed: https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom
    * Checked that there is not already an Atom package that provides the described functionality: https://atom.io/packages

### Description

Autocompletion menu not appearing for CSS values after autocompleting a property.

### Steps to Reproduce

With some CSS file, enter:

```css
.div {
}
```

1. In the .div selector, start typing a property e.g. type `marg`.
2. Select `margin` from the autocomplete menu.

**Expected behavior:** [What you expect to happen]

Expect autocomplete menu popup for `margin` properties e.g. `auto`, `inherit`.

**Actual behavior:** [What actually happens]

No menu popup.

**Reproduces how often:** [What percentage of the time does it reproduce?]

100%

### Versions

```
atom -v
Atom    : 1.22.1
Electron: 1.6.15
Chrome  : 56.0.2924.87
Node    : 7.4.0

apm -v
apm  1.18.8
npm  3.10.10
node 6.9.5 x64
python 2.7.10
git 2.14.3
```

macOS 10.12.6

### Additional Information

Any additional information, configuration or data that might be necessary to reproduce the issue.

GIF from @KeooR:

![ac-bug](https://user-images.githubusercontent.com/13817999/34263706-a12b1272-e670-11e7-8913-8db878beb224.gif)

Worked in 1.22.1:

![ac-1.22.1](https://user-images.githubusercontent.com/734194/34588471-cb7c2a2c-f172-11e7-894a-ac1db8a2fabe.gif)

---

*Original report by @Andronachi-Marian*

### Description
If i type 'marg' and ENTER the autocomplete not show after is inserted 'margin: '.
If i press CTRL+SPACE its show all the css properties "list-style-image, width, margin, etc".

Works only if i remove the space after 'margin:' and works and CTRL+SPACE.

**Expected behavior:** To show the autocoplete if is one or more spaces after 'margin:'.
**Actual behavior:** Not working
**Reproduces how often:** 5s

### Versions

Atom    : 1.23.1
Electron: 1.6.15
Chrome  : 56.0.2924.87
Node    : 7.4.0
Package: 0.17.4

Thanks.

**Comment by AlexKhalaimov** [comment]
Have the same bug.

For example:  I'm going to  type "text-transform: capitalize"  property , I type "text-" and then choose "text-transform" in help-box, press ENTER and help-box with all css properties of "text-transform" is not showed . I have only "text-transform:" typed in my css code.
Sorry for my bad english.


System: Win 7*64
Atom: 1.23.1
Atom-autocomplete: 0.7.14

Thanks

**Comment by ghost** [comment]
I encounter the same problem, I made a video to illustrate.

![autocomplete-css not working](https://user-images.githubusercontent.com/13817999/34263706-a12b1272-e670-11e7-8913-8db878beb224.gif)

I hope it could help.

**Comment by rsese** [comment]
Thanks for the report - I can reproduce with 1.23.2 on macOS 10.12.6.  it looks like this worked in 1.22.1 though:

![ac-css-1 22 1](https://user-images.githubusercontent.com/734194/34588471-cb7c2a2c-f172-11e7-894a-ac1db8a2fabe.gif)

**Comment by winstliu** [comment]
This seems to be an upstream autocomplete-plus regression as there have been no code changes in autocomplete-css for a few months now.

**Comment by winstliu** [comment]
Looks like either https://github.com/atom/autocomplete-css/blob/53079283bfcd671c31e0b8e013b0c83279ff3608/lib/provider.coffee#L48 isn't firing or something changed with how autocomplete-plus reacts to the `autocomplete-plus:activate` command.

**Comment by rsese** [comment]
Thanks @50Wliu - I was about to copy this over to https://github.com/atom/autocomplete-plus/ but I found that this looks fixed in the latest beta `1.24.0-beta3` released today.

I'm going to go ahead and close this but if I misunderstood anything or if folks still see a problem with the latest beta we can definitely re-open.
